views/book.html

```diff
@@ -1,5 +1,5 @@
 <!-- Book Page -->
-{{ block "book-page" . }}
+{{ define "book-page" }}
```

Comment:
I just realized that we're not even using this component. When you rebase and get all the new stuff into this PR then remove this file or replace this with the result-card